core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -251,6 +255,10 @@ public int main(List<String> args, Locale locale, InputStream stdin, PrintStream
             stderr.println("Bad Credentials. Search the server log for "+id+" for more details.");
             return -1;
         } catch (Exception e) {
+            final String errorMsg = String.format("Unexpected exception occurred during performing %s command: %s",
+                    getName(), e.getMessage());
+            stderr.println(errorMsg);
+            LOGGER.log(Level.WARNING, errorMsg);
```

Comment:
No, this part of code is executed on master, the stderr is sent to the client, but log stays on the server where the destination depends on the log configuration of Jenkins master - common cfg. - final target is server log or the console.